src/inverseRobot.js

```diff
@@ -6,8 +6,22 @@
  * @return {object}
 */
 
+const objLength = (obj) => {
+  return Object.keys(obj).length;
+};
+
 function inverseRobot(robot) {
-  // write code here
+  const naturalRobot = {};
+
+  for (const key in robot) {
+    naturalRobot[robot[key]] = key;
+  }
+
+  if (objLength(naturalRobot) === objLength(robot)) {
```

Comment:
Better will be to check in the loop if your new robot already contains such key
P.S. Take a look on checklist